Next.js production build fail

Build fails when trying to build using github actions.
Error:
```
#16 [builder 4/4] RUN yarn build
#16 0.335 yarn run v1.22.19
#16 0.361 $ next build
#16 1.007 Attention: Next.js now collects completely anonymous telemetry regarding usage.
#16 1.008 This information is used to shape Next.js' roadmap and prioritize features.
#16 1.008 You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
#16 1.008 https://nextjs.org/telemetry
#16 1.008 
#16 1.105    ▲ Next.js 14.1.4
#16 1.105 
#16 1.175    Creating an optimized production build ...
#16 6.687 Failed to compile.
#16 6.687 
#16 6.688 ./src/components/parts/squareButton.tsx
#16 6.688 Module not found: Can't resolve './SquareButton.module.css'
#16 6.688 
#16 6.688 https://nextjs.org/docs/messages/module-not-found
#16 6.688 
#16 6.688 Import trace for requested module:
#16 6.688 ./src/components/sections/DifferentEffectsSection.tsx
#16 6.688 
#16 6.707 
#16 6.708 > Build failed because of webpack errors
#16 6.727 error Command failed with exit code 1.
#16 6.727 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
#16 ERROR: process "/bin/sh -c yarn build" did not complete successfully: exit code: 1
------
 > [builder 4/4] RUN yarn build:
6.688 
6.688 https://nextjs.org/docs/messages/module-not-found
6.688 
6.688 Import trace for requested module:
6.688 ./src/components/sections/DifferentEffectsSection.tsx
6.688 
6.707 
6.708 > Build failed because of webpack errors
6.727 error Command failed with exit code 1.
6.727 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Dockerfile:
```
FROM node:20.10.0-alpine AS base

# Install dependencies only when needed
FROM base AS deps
# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
RUN apk add --no-cache libc6-compat
WORKDIR /app

# Install dependencies based on the preferred package manager
COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
RUN \
  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
  elif [ -f package-lock.json ]; then npm ci; \
  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i --frozen-lockfile; \
  else echo "Lockfile not found." && exit 1; \
  fi


# Rebuild the source code only when needed
FROM base AS builder
WORKDIR /app
COPY --from=deps /app/node_modules ./node_modules
COPY . .

# Next.js collects completely anonymous telemetry data about general usage.
# Learn more here: https://nextjs.org/telemetry
# Uncomment the following line in case you want to disable telemetry during the build.
# ENV NEXT_TELEMETRY_DISABLED 1

RUN yarn build

# If using npm comment out above and use below instead
# RUN npm run build

# Production image, copy all the files and run next
FROM base AS runner
WORKDIR /app

ENV NODE_ENV production
# Uncomment the following line in case you want to disable telemetry during runtime.
# ENV NEXT_TELEMETRY_DISABLED 1

RUN addgroup --system --gid 1001 nodejs
RUN adduser --system --uid 1001 nextjs

COPY --from=builder /app/public ./public

# Set the correct permission for prerender cache
RUN mkdir .next
RUN chown nextjs:nodejs .next

# Automatically leverage output traces to reduce image size
# https://nextjs.org/docs/advanced-features/output-file-tracing
COPY --from=builder --chown=nextjs:nodejs /app/.next/standalone ./
COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static

USER nextjs

EXPOSE 8084

ENV PORT 8084
# set hostname to localhost
ENV HOSTNAME "0.0.0.0"

# server.js is created by next build from the standalone output
# https://nextjs.org/docs/pages/api-reference/next-config-js/output
CMD ["node", "server.js"]
```

--- a/src/components/parts/SquareButton.module.css
+++ b/src/components/parts/SquareButton.module.css
@@ -1,0 +1,88 @@
+.squareButton {
+    width: 117px;
+    height: 117px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    border-radius: 20px;
+    background: #262626;
+    /* website image */
+    box-shadow: 0 0 75px 0 rgba(0, 0, 0, 0.50), 0px 4px 60px 0px rgba(0, 0, 0, 0.30) inset;
+}
+
+.iconBlue {
+    color: var(--Accent_1);
+    font-size: 55px!important;
+}
+.iconBlack {
+    color: black;
+    font-size: 55px!important;
+}
+
+@media screen and (max-width: 1439px) {
+    .squareButton {
+        width: 105px;
+        height: 105px;
+    }
+    .iconBlue {
+        font-size: 50px!important;
+    }
+    .iconBlack {
+        font-size: 50px!important;
+    }
+}
+
+@media screen and (max-width: 1280px) {
+    .squareButton {
+        width: 90px;
+        height: 90px;
+    }
+    .iconBlue {
+        font-size: 40px!important;
+    }
+    .iconBlack {
+        font-size: 40px!important;
+    }
+}
+
+@media screen and (max-width: 1000px) {
+    .squareButton {
+        width: 75px;
+        height: 75px;
+        border-radius: 10px;
+    }
+    .iconBlue {
+        font-size: 30px!important;
+    }
+    .iconBlack {
+        font-size: 30px!important;
+    }
+}
+
+@media screen and (max-width: 768px) {
+    .squareButton {
+        width: 90px;
+        height: 90px;
+    }
+    .iconBlue {
+        font-size: 40px!important;
+    }
+    .iconBlack {
+        font-size: 40px!important;
+    }
+}
+
+@media screen and (max-width: 500px) {
+    .squareButton {
+        width: 60px;
+        height: 60px;
+    }
+    .iconBlue {
+        font-size: 25px!important;
+    }
+    .iconBlack {
+        font-size: 25px!important;
+    }
+}

